README.md

```diff
@@ -40,6 +40,8 @@ In the `*Ebuku*` buffer, the following bindings are available:
 
 * <kbd>RET</kbd> - Open the bookmark at point in a browser (`ebuku-open-url`).
 
+* <kbd>C</kbd> - Copy the bookmark at point to kill ring (`ebuku-copy-url`).
```

Comment:
The README is generated from the source by using `el2markdown`, so just remove this change.